main.js

```diff
@@ -0,0 +1,51 @@
+const { app, BrowserWindow } = require("electron");
+const { spawn } = require("child_process");
+const path = require("node:path");
+
+const startFlaskServer = () => {
+  const device_xml = path.join(__dirname, "backend/etc/device.xml");
+  const apiServer = spawn(`python`, ["backend/restapi_server.py", device_xml]);

```

Comment:
Would it be great if we construct the absolute path of backend/restapi_server.py like we are doing in the above line for device.xml? In deployment mode, we can place the restapi_server.py file anywhere and Python will find it. 
Also, can we use python3 instead of Python as on Linux it is mostly python3, or maybe with whatever exe name of Python is available, use it?

Reply:
What about Windows, I tried "python3" Windows doesn't recognize.